Adjust the text used in various man pages to use the Solaris specific
man page section or remove references to commands that are't relevant
to Solaris.

This patch has not been passed upstream yet. It needs to be reworked into
a format that would be acceptable.

--- man/mkfifo.x.orig	2014-07-30 06:49:11.199126893 -0700
+++ man/mkfifo.x	2014-07-30 06:53:52.101716577 -0700
@@ -3,4 +3,4 @@
 [DESCRIPTION]
 .\" Add any additional description here
 [SEE ALSO]
-mkfifo(3)
+mkfifo(3C)
--- man/mktemp.x.orig	2014-07-30 06:49:11.203140097 -0700
+++ man/mktemp.x	2014-07-30 06:54:11.141190540 -0700
@@ -3,4 +3,4 @@
 [DESCRIPTION]
 .\" Add any additional description here
 [SEE ALSO]
-mkstemp(3), mkdtemp(3), mktemp(3)
+mkstemp(3C), mkdtemp(3C), mktemp(3C)
--- man/printf.x.orig	2014-07-30 06:49:11.206916592 -0700
+++ man/printf.x	2014-07-30 06:54:19.607330057 -0700
@@ -3,4 +3,4 @@
 [DESCRIPTION]
 .\" Add any additional description here
 [SEE ALSO]
-printf(3)
+printf(3C)
--- man/pwd.x.orig	2014-07-30 06:49:11.210693423 -0700
+++ man/pwd.x	2014-07-30 06:54:31.497670121 -0700
@@ -3,4 +3,4 @@
 [DESCRIPTION]
 .\" Add any additional description here
 [SEE ALSO]
-getcwd(3)
+getcwd(3C)
--- man/rm.x.orig	2014-07-30 06:49:11.214453258 -0700
+++ man/rm.x	2014-07-30 06:54:55.003442272 -0700
@@ -28,4 +28,4 @@
 not affirmative, the file is skipped.
 .SH OPTIONS
 [SEE ALSO]
-unlink(1), unlink(2), chattr(1), shred(1)
+unlink(1), unlink(2), shred(1)
--- man/sleep.x.orig	2014-07-30 06:49:11.217060981 -0700
+++ man/sleep.x	2014-07-30 06:55:05.302644446 -0700
@@ -3,4 +3,4 @@
 [DESCRIPTION]
 .\" Add any additional description here
 [SEE ALSO]
-sleep(3)
+sleep(3C)
--- man/tac.x.orig	2014-07-30 06:49:11.219413108 -0700
+++ man/tac.x	2014-07-30 06:55:43.098711014 -0700
@@ -3,4 +3,3 @@
 [DESCRIPTION]
 .\" Add any additional description here
 [SEE ALSO]
-\fBrev\fP(1)
--- man/truncate.x.orig	2014-07-30 06:49:11.224283357 -0700
+++ man/truncate.x	2014-07-30 06:56:07.280167859 -0700
@@ -3,4 +3,4 @@
 [DESCRIPTION]
 .\" Add any additional description here
 [SEE ALSO]
-dd(1), truncate(2), ftruncate(2)
+dd(1), truncate(3C), ftruncate(3C)
